recipes/RECOLA/run_all.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+
+def main():
```

Comment:
This is usually part of one train.py file - which also contains the expBrain.
Please compare:
https://github.com/speechbrain/speechbrain/tree/develop/recipes/IEMOCAP/emotion_recognition
// I reference it again as it seems a recipe with alike task.